CONTRIBUTING.md

```diff
@@ -97,3 +97,33 @@ If you make a (significant) contribution to ESMValCore, please add your name to
 
     pip install cffconvert
     cffconvert --ignore-suspect-keys --outputformat zenodo --outfile .zenodo.json
+
+## How to make a release
+
+To make a new release of the package, follow these steps
+
+### 1. Check that the nightly build on CircleCI was successful
+
+Check the `nightly` [build on CircleCI](https://circleci.com/gh/ESMValGroup/ESMValTool/tree/master).
+All tests should pass before making a release.
+
+### 2. Make a pull request to increase the version number
+
+The version number is stored in esmvalcore/_version.py, meta.yaml, CITATION.cff.
```

Comment:
```suggestion
The version number is stored in `esmvalcore/_version.py`, `meta.yaml`, `CITATION.cff`.
```
Otherwise `_` formats the following text in _italic_